log-effectful.cabal

```diff
@@ -41,7 +41,7 @@ library
     , aeson
     , base            <=4.17
     , bytestring
-    , effectful-core  ^>=1.0.0.0
+    , effectful-core  ^>=2.1.0.0
```

Comment:
Why such a restrictive lower-bound?  Why not `effectful-core >=1.0 && <2.2`, as in time-effectful?

Reply:
No reason really, I've fixed the lower bounds as per your suggestion.